pkgs/servers/microbin/default.nix

```diff
@@ -1,11 +1,13 @@
-{ lib
-, rustPlatform
+{ darwin
 , fetchFromGitHub
-, pkg-config
+, fetchpatch
+, lib
 , oniguruma
 , openssl
+, pkg-config
+, rustPlatform
 , stdenv
-, darwin
+, noPatches ? false
```

Comment:
Equivalent functionality to setting `dontPatch = true;`, and it prolly makes more sense to just have the maintainers decide whether the patches are good & worth applying in general or not.

```suggestion
```

Reply:
ok then, thanks, I'll get rid of that switch